usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -364,13 +389,15 @@ def equal_datasets(psql_data: List[Dict[str, Any]], spark_data: List[Dict[str, A
     return datasets_match
 
 
-def _verify_delta_table_loaded_to_delta(
+def verify_delta_table_loaded_to_delta(
     spark: SparkSession,
     delta_table_name: str,
     s3_bucket: str,
     alt_db: str = None,
     alt_name: str = None,
     load_command="load_table_to_delta",
+    dummy_data: List[Dict[str, Any]] = None,
```

Comment:
I don't think I saw `dummy_data` passed into this method anywhere? If not being passed over is there any reason why this couldn't be defined in the method body, allowing that nested conditional to be removed.

Reply:
I did not realize that this was also used by `test_load_to_from_delta_subawards.py`. After speaking with Daniel I am happy with the change and closing this conversation.